bundles/org.openhab.binding.ojelectronics/src/main/resources/OH-INF/update/thing-update.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="ojelectronics:owd5">
+		<instruction-set targetVersion="1">
+			<update-channel id="vacationBeginDay">
+				<type>ojelectronics:vacationBeginDay</type>
+			</update-channel>
+			<update-channel id="vacationEndDay">
+				<type>ojelectronics:vacationEndDay</type>
+			</update-channel>
```

Comment:
Same here, is it better or not to include what is changed (itemType / description).